src/pyenphase/updaters/api_v1_production.py

```diff
@@ -28,7 +28,9 @@ async def probe(
                 "Production endpoint not found at %s: %s", URL_PRODUCTION_V1, e
             )
             return None
-        if all(value == 0 for value in response.values()):
+        if all(
+            value == 0 for key, value in response.items() if key.startswith("wattHours")
```

Comment:
This could use a comment explaining why `wattsNow` is not considered

Reply:
Added more comment in code.